search/factory.py

```diff
@@ -61,13 +60,7 @@ def create_api_web_app() -> Flask:
     index.SearchSession.init_app(app)
 
     Base(app)
-    # auth.Auth(app)
```

Comment:
You found that auth was not in use? That's interesting.

Reply:
Please remove the auth lines instead of commenting it out?

Reply:
Yeah, this does delete the comments ;)
Looks like Martin [disabled it 3 years ago](https://github.com/arXiv/arxiv-search/commit/1e3424e967d84be62fcae00ef41b3c097d2de9a6)